draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -155,36 +157,38 @@ An ICE agent that supports mDNS candidates MUST support the situation where the
 In this case, the ICE agent MUST take exactly one of the resolved IP addresses and ignore the others.
 The ICE agent SHOULD, if available, use the first IPv6 address resolved, otherwise the first IPv4 address.
 
-### Handling of Peer-Reflexive Remote Candidate
-
-A peer-reflexive remote candidate could be learned and constructed from the
-source transport address of the STUN Binding request as an ICE connectivity
-check. The peer-reflexive candidate could share the same address as a remote mDNS
-ICE candidate whose name is
-in the process of being resolved. In addition to the elimination procedure
-of redundant candidates defined in Section 5.1.3 of {{RFC8445}}, which could
-remove constructed peer-reflexive remote candidates, the address of any existing
-peer-reflexive remote candidate should not be exposed to Web applications by ICE
-agents that implement this proposal, as detailed in Section {{#guidelines}}.
-
-Privacy Guidelines {#guidelines}
-============
-
-APIs leaking IP addresses
-----------------------------
-
-When there is no user consent, the following filtering should be done to prevent private IP address leakage:
-
-1. ICE candidates with an IP address are not exposed as ICE candidate events.
-
-2. Server reflexive ICE candidate raddr field is set to 0.0.0.0 and rport to 0.
-
-3. SDP does not expose any a=candidate line corresponding to an ICE candidate which contains an IP address.
```

Comment:
This item need to be rephrased to limit the exposure of IP for mDNS candidates. I think the meaning has changed after we changed the original "ICE host candidate" to "ICE candidate." This item is also not in the revision of this PR. Though I feel it should be part of the definition of mDNS candidate (and its signaling), do we also want to keep it in {#privacy}?

Reply:
I think this should be covered by the #gathering section - thoughts? Or do we want to reiterate this for getLocalDescription/getRemoteDescription?

Reply:
> I think this should be covered by the #gathering section

Agreed, it reads to me now step 6 in the #gathering section should have covered it. It is not specific to SDP or local description, but SDP is rather an example protocol of exchange in 8445 and local description reads to me more WebRTCSpec. I am adding a few words to refer to the candidate exchange in 8445. Let me know if it reads okay.

Reply:
Thought about it a bit more, and not sure citing 8445 makes sense here. We're not actually sending the candidate to the remote side, as noted in 8445, we're just giving it to the web app here.

Reply:
As a result, I left step 6 mostly as-is.

Reply:
OK, will add a note on this.

Reply:
Added some new text, PTAL.